aclk/README.md

```diff
@@ -1,10 +1,120 @@
 <!--
 ---
 title: "Agent-cloud link (ACLK)"
+description: "The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. 
+The ACLK uses MQTT over secure websockets to create, persist, and encrypt the connect from end-to-end, and enable the 
+features found in Netdata Cloud."
+date: 2020-03-16
 custom_edit_url: https://github.com/netdata/netdata/edit/master/aclk/README.md
 ---
 -->
 
 # Agent-cloud link (ACLK)
 
-This is the agent cloud link (ACLK) information file
\ No newline at end of file
+The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. The ACLK uses
+[MQTT](https://en.wikipedia.org/wiki/MQTT) over secure websockets to create, persist, and encrypt the connect from
+end-to-end, and enable the features found in Netdata Cloud.
+
+The ACLK is a vital component of the entirely redesigned and re-architected Netdata Cloud.
+
+_No data is exchanged with Netdata Cloud until you claim a node._ By claiming a node, you opt-in to sending data from
+your agent to Netdata Cloud via the ACLK.
+
+## Claim nodes with the ACLK
```

Comment:
This overlaps with claim/README.md - should we merge docs?

Reply:
Good call. I'm going to put some of this over in `claim/README.md` and just link from one doc to the other.